packages/client/src/generation/TSClient/common.ts

```diff
@@ -18,6 +22,24 @@ const {
   Decimal,
   objectEnumValues
 } = require('${runtimeDir}/${runtimeName}')
+`
+    : runtimeName === 'edge'
+    ? `import {
+  Decimal,
+  decompressFromBase64,
+  empty,
+  getPrismaClient,
+  join,
+  NotFoundError,
+  objectEnumValues,
+  PrismaClientInitializationError,
+  PrismaClientKnownRequestError,
+  PrismaClientRustPanicError,
+  PrismaClientUnknownRequestError,
+  PrismaClientValidationError,
+  raw,
+  sqltag,
+} from "${runtimeDir}/${runtimeName}.js"
```

Comment:
Changed this to ES Module syntax and also added `.js` extension here. This is necessary for Deno, and this also looks ok for cloudflare workers because webpack can resolve this.

Reply:
(https://github.com/prisma/prisma/pull/14386/files#r930359993)